src/components/sheet/SlackSheet.js

```diff
@@ -92,11 +92,13 @@ export default forwardRef(function SlackSheet(
     showBlur,
     testID,
     removeClippedSubviews = false,
+    yPosition: givenYPosition,
     ...props
   },
   ref
 ) {
-  const yPosition = useSharedValue(0);
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const yPosition = givenYPosition || useSharedValue(0);
```

Comment:
oh helllllllll no :)

Reply:
@osdnk

Reply:
actually I understand the reasoning so we might have no other choice